{
  "api": "indices.rollover",
  "file": "/test/free/data_stream/210_rollover_failure_store.yml",
  "name": "Rolling over a failure store on a data stream without the failure store enabled should work",
  "origin": "yaml",
  "request": {
    "args": {
      "alias": "other-data-stream-for-rollover::failures"
    }
  },
  "response": {
    "headers": {
      "content-length": "246",
      "content-type": "application/json",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "acknowledged": true,
      "conditions": {
      },
      "dry_run": false,
      "lazy": false,
      "new_index": ".fs-other-data-stream-for-rollover-2025.10.31-000003",
      "old_index": ".fs-other-data-stream-for-rollover-2025.10.31-000002",
      "rolled_over": true,
      "shards_acknowledged": true
    },
    "statusCode": 200
  }
}